subsys/nrf_security/src/drivers/CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ if(CONFIG_MBEDTLS_PSA_CRYPTO_C)
     add_subdirectory(nrf_oberon)
   endif()
 
+  if(CONFIG_PSA_NEED_CRACEN_TRNG_DRIVER)
```

Comment:
I would propose to move this code under the nrf_oberon Cmakefile because it belongs to this level I think since it is not a PSA driver exactly.